src/sql/migrations/20230907104002_cell_metadata_tables.js

```diff
@@ -0,0 +1,107 @@
+const {
+  CELL_METADATA,
+} = require('../../config/bucketNames');
```

Comment:
I think you might need to mimic this imported value in iac (since the migration files are copied over there)